src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -586,6 +599,9 @@ void AttitudeEstimatorQ::task_main()
 
 		ctrl_state.yaw_rate = _lp_yaw_rate.apply(_rates(2));
 
+		/* Airspeed - take airspeed measurement directly here as no wind is estimated */
+		ctrl_state.airspeed = _airspeed.true_airspeed_m_s;
```

Comment:
We should be switching to indicated airspeed, as per https://github.com/PX4/Firmware/issues/3261